.travis.yml

```diff
@@ -11,17 +11,19 @@ notifications:
   email:
```

Comment:
i would keep the docker files and the travis changes in separate PRs, to reduce the issues of circular dependencies you talked about.